emuu-client/src/components/UserProfile/Feeds.js

```diff
@@ -174,7 +248,67 @@ function Feeds({ setVideo }) {
                     </CardContent>
                   </Card>
                 ))}
+
+              {topVideos &&
+                sort == "Most Viewed" &&
+                topVideos.map((video) => (
+                  <Card sx={{ maxWidth: 380, height: 400 }}>
+                    <CardMedia component="img" image={video.ThumbnailUrl} />
+                    <CardContent>
+                      <CardHeader
+                        avatar={
+                          <Avatar
+                            sx={{ width: 60, height: 60 }}
+                            src={ProfilePic}
+                          ></Avatar>
+                        }
+                        title={
+                          <Typography
+                            variant="body2"
+                            color="text.primary"
+                            fontWeight="bold"
+                            fontSize="20px"
+                          >
+                            <Link to="/video">
+                              <span
+                                onClick={() => {
+                                  setVideo(video);
+                                }}
+                              >
+                                {video.Title}
+                              </span>
+                            </Link>
+                          </Typography>
+                        }
+                      />
+
+                      <div className="videoInfo">
+                        <Typography
+                          variant="body2"
+                          color="text.secondary"
+                          fontWeight="medium"
+                          fontSize="18px"
+                        >
+                          {" "}
+                          {video.Username} &ensp;&ensp;&ensp;&ensp;&ensp;
+                          {video.Likes} Likes &#x2022; {video.Views} Views
```

Comment:
Fix this after merge, I updated it so likes and username were on different lines but it seems to be overwritten.